### DISCUSSION TITLE
Build Process - Behind the Scenes ?

### DISCUSSION THREAD

**Discussion by uneknown** [vercel/community#804] [Help] [answered]
I use Vercel on a daily basis. I love it! But there is something that's bugging me. There are no resources provided on how the build process works behind the scenes! For stuff that I use daily, I like to know what happens in the background 😄 What is Vercel using to provision the build environments? How are they isolated? What does the general architecture look like?

It would be great if someone could outline this to me!

**Comment by TomGalla11** [comment]
Good question! Would appreciate an answer from @vercel!

**Comment by amyegan** [comment, marked as answer]
Hi! We don't currently have a document that outlines our entire build process, but that sounds like a great idea and we're putting it on our list of things to add. Thank you for letting us know that this is something you want to see! 

At the moment, you can find some relevant information about the build process in our documentation and blog:
- [Engineering Blog](https://vercel.com/blog/category/engineering)
- [Troubleshoot a Build](https://vercel.com/docs/concepts/deployments/troubleshoot-a-build)
- [Configure a Build](https://vercel.com/docs/concepts/deployments/configure-a-build)
- [Changelog](https://vercel.com/changelog)
